Makefile

```diff
@@ -42,6 +42,7 @@ TOITPKG_BIN = $(BIN_DIR)/toitpkg$(EXE_SUFFIX)
 TOITLSP_BIN = $(BIN_DIR)/toitlsp$(EXE_SUFFIX)
 TOITVM_BIN = $(BIN_DIR)/toitvm$(EXE_SUFFIX)
 TOITC_BIN = $(BIN_DIR)/toitc$(EXE_SUFFIX)
+VERSION_FILE = $(BIN_DIR)/VERSION
```

Comment:
Is this automatically packaged up as part of releases?

Reply:
yes

Reply:
every file in sdk will be available in the tar.